components/invitation_modal/invitation_modal_guest_step/invitation_modal_guests_step.jsx

```diff
@@ -180,8 +176,8 @@ class InvitationModalGuestsStep extends React.PureComponent {
     };
 
     componentDidMount() {
-        if (!this.props.analytics) {
-            this.props.actions.getStandardAnalytics();
+        if (isEmpty(this.props.subscriptionStats)) {
+            this.props.actions.getSubscriptionStats();
```

Comment:
Missed in the initial pass that we should only call `getSubscriptionStats` if it's in cloud. Also, if not in cloud, could `props.subscriptionStats` be null? If so, let's add a null check when necessary to avoid issues

Reply:
Hi @marianunez thanks for the review. I have updated the code to make the checks. Here's the redux PR https://github.com/mattermost/mattermost-redux/pull/1358 that needs approval before this one can be merged.